### PR TITLE
update index

### PR DESCRIPTION
**Pull request by ainanjh**

### CODE REVIEW

**Review by juasfan — CHANGES_REQUESTED** [review]
Nice overall i just need to comment lmao

File: index.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
   <meta name="description" content="" />
   <meta name="author" content="" />
-  <title>Online Library Management System | </title>
+  <title>Online Library Management System </title>
```
Comment (juasfan) [inline index.php:46]: eh

**Review by juasfan — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (ainanjh) [reply index.php:46]: why

Comment (juasfan) [reply index.php:46]: idk sisturs

Comment (juasfan) [reply index.php:46]: is this how it is done

**Review by juasfan — COMMENTED** [review]
heh

**Pull request merged**